examples/todo-intermediate/CHANGELOG.md

```diff
@@ -0,0 +1,512 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="0.12.5"></a>
+## [0.12.5](https://github.com/strongloop/loopback-next/compare/@loopback/example-todo@0.12.4...@loopback/example-todo@0.12.5) (2018-06-28)
```

Comment:
Please remove the CHANGELOG, todo-intermediate should start with clean history IMO.